File: tvn-cosine.ai/tvn-cosine.ai/logic/fol/Quantifiers.cs
namespace tvn.cosine.ai.logic.fol
{
    public class Quantifiers
    {
        public static final string FORALL = "FORALL";
	public static final string EXISTS = "EXISTS";

	public static bool isFORALL(string quantifier)
        {
            return FORALL.Equals(quantifier);
        }

        public static bool isEXISTS(string quantifier)
        {
            return EXISTS.Equals(quantifier);
        }
    }
}

File: tvn-cosine.ai/tvn-cosine.ai.v2/logic/basic/firstorder/Quantifiers.java
namespace aima.core.logic.basic.firstorder;

/**
 * @author Ciaran O'Reilly
 * @author Anurag Rai
 */
public class Quantifiers {
	public static final String FORALL = "FORALL";
	public static final String EXISTS = "EXISTS";

	public static boolean isFORALL(String quantifier) {
		return FORALL.Equals(quantifier);
	}

	public static boolean isEXISTS(String quantifier) {
		return EXISTS.Equals(quantifier);
	}
}
